rpcd/playbooks/roles/rpc_maas/defaults/main.yml

```diff
@@ -352,6 +352,20 @@ cpu_memory_checks_list:
 disk_utilisation_checks_list:
   - { name: "disk_utilisation", group: "hosts" }
 
+rsyslogd_process_names:
+  - rsyslogd
+
+elasticsearch_process_names:
+  - elasticsearch
```

Comment:
So I think we've found that this is not the process name. The process name is "java". ;)